Description: Optimization changes the name of a single variable.
Author: Andrius Merkys <merkys@debian.org>
--- a/test/requirejs_test.js
+++ b/test/requirejs_test.js
@@ -8,7 +8,7 @@
 
     test.expect(1);
 
-    expect = 'define("hello",[],function(){return"hello"}),define("world",[],function(){return"world"}),require(["hello","world"],function(e,t){console.log(e,t)}),define("project",function(){});';
+    expect = 'define("hello",[],function(){return"hello"}),define("world",[],function(){return"world"}),require(["hello","world"],function(e,n){console.log(e,n)}),define("project",function(){});';
     result = grunt.file.read('tmp/requirejs.js');
     test.equal(expect, result, 'should optimize javascript modules with requireJS');
 
@@ -20,7 +20,7 @@
 
     test.expect(1);
 
-    expect = 'define("hello",[],function(){return"hello"}),define("world",[],function(){return"world"}),require(["hello","world"],function(e,t){console.log(e,t)}),define("project",function(){});';
+    expect = 'define("hello",[],function(){return"hello"}),define("world",[],function(){return"world"}),require(["hello","world"],function(e,n){console.log(e,n)}),define("project",function(){});';
     result = grunt.file.read('tmp/requirejs-template.js');
     test.equal(expect, result, 'should process options with template variables.');
 
